### DISCUSSION TITLE
Thank you very much, fantastic project

### DISCUSSION THREAD

**Discussion by lchris789** [landrix/Zint-Barcode-Generator-for-Delphi#28] [General]
Just used it tu generate a datamatrix C40 encoded barcode ! works great

Compiled with 0 errors in Delphi 11.1

just took a little time to figure out how to force C40

for those who wonder, encoding /compression format is choosen automaticaly in
unit zint_dmatrix function look_ahead_test
To force, at the end just add  Result := DM_C40;  instead of Result := best_scheme;
     
best regards

![DelphiZintDataMatrix](https://user-images.githubusercontent.com/48279740/160156311-af373166-e934-40db-a0a7-53d6910dfabe.png)

**Comment by landrix** [comment]
ok,
but how can I respect the other schemes that determine best_scheme. Maybe something else needs to be set to force DM_C40?
